static/js/controllers/inbox.js

```diff
@@ -148,6 +148,16 @@ require('moment/locales');
         $scope.items = contacts || [];
       };
 
+      $scope.removeContact = function(contact) {
```

Comment:
I had a reason for putting the method in this particular controller, but I'm not sure if it's a good reason... there is another method in `inbox.js` called `removeDeletedContacts()`, which I initially tried to call to do the dirty work for me.  Sadly it didn't really work.  If there is a more suitable place for this function to go (e.g. in `ContactsCtrl`) then I will happily move it.